tracer/src/Datadog.Trace.Tools.dd_dotnet/Checks/Resources.cs

```diff
@@ -57,6 +57,8 @@ internal static class Resources
 
         public static void ResetChecks() => _checkNumber = 1;
 
+        public static string EnableDiagnosticsSet(string key) => $"The environment variable {key} is set to 0, which disables profiling.";
```

Comment:
`COMPlus_EnableDiagnostics=0` isn't _actually_ a problem unless we're running under .NET 8 🤔 Do we care? Should we mention that it's ok if you're running <.NET 8?

Reply:
I think it brings nothing and just increase the risks of false-negatives 😛

Reply:
Sure, fair enough